src/documents/views/ReviewListView.js

```diff
@@ -0,0 +1,190 @@
+    );
+
+  if (error)
+    return (
+      <Container as={Segment} basic>
+        <Helmet>
+          <title>
+            {`KF Data Tracker - Study documents - Error ${
+              study ? 'for ' + study.kfId : null
+            }`}
+          </title>
+        </Helmet>
+        <Message
+          negative
+          icon="warning circle"
+          header="Error"
+          content={error.message}
+        />
+      </Container>
+    );
+
+  return (
+    <Grid as={Segment} basic container columns={1}>
+      <Helmet>
+        <title>
+          {`KF Data Tracker - Study documents ${
+            study ? 'for ' + study.name : null
+          }`}
+        </title>
+      </Helmet>
+      <Grid.Row>
+        <Grid.Column width={4}>
+          <h2>Study Reviews</h2>
+        </Grid.Column>
+        <Grid.Column width={12} textAlign="right">
+          {allowAddReview && (
+            <Button
+              compact
+              color="teal"
+              floated="right"
+              size="large"
+              icon="check"
+              labelPosition="left"
+              content="Start New Review"
+              as="label"
+              onClick={() => history.push(`/study/${study.kfId}/start-review`)}
+            />
+          )}
+        </Grid.Column>
+      </Grid.Row>
+      <Grid.Row>
+        <Grid.Column width={16}>
+          {reviews.length > 0 ? (
+            <List relaxed="very">
+              {reviews.map(({node}) => (
+                <List.Item key={node.kfId}>
+                  <Popup
+                    inverted
+                    position="left center"
+                    content={reviewStatus[node.state].text}
+                    trigger={
+                      <List.Icon
+                        name={reviewStatus[node.state].icon}
+                        color={reviewStatus[node.state].color}
+                        size="big"
+                        verticalAlign="middle"
+                      />
+                    }
+                  />
+                  <List.Content>
+                    <List.Header
+                      as="a"
+                      onClick={() =>
+                        history.push(
+                          `/study/${study.kfId}/reviews/${node.kfId}`,
+                        )
+                      }
+                    >
+                      {node.name.substring(0, 100)}
+                      {node.name.length > 100 && '...'}
+                      <small className="text-normal text-grey ml-10">
+                        {node.kfId}
+                      </small>
+                      <small className="text-normal text-grey ml-10">
+                        <TimeAgo
+                          date={node.createdAt}
+                          live={false}
+                          title={longDate(node.createdAt)}
+                        />
+                      </small>
+                    </List.Header>
+                    <List.Description as="a">
```

Comment:
Including the description here may be a little awkward because the formatting can be anything. 
Another idea may be to include the number of documents in the review.